setup.cfg

```diff
@@ -43,16 +43,18 @@ python_requires = >=3.8
 include_package_data = True
 install_requires =
     appdirs>=1.4.4
+    app-model>=0.0.6
     cachey>=0.2.1
     certifi>=2018.1.18
     dask[array]>=2.15.0,!=2.28.0  # https://github.com/napari/napari/issues/1656
     imageio>=2.5.0,!=2.11.0
+    in-n-out>=0.1.3
```

Comment:
Does this need to be here? It looks like it's used in app-model but not directly by napari.

Reply:
it was added per reviewer request in #4847 ... but at that time, I was directly importing and using it in napari.  Now I'm not, and will remove it again